Mlem/App/Utility/Extensions/Views/View Modifiers/View+PaletteBorder.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  View+PaletteBorder.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-10-10.
+//
+
+import SwiftUICore
+
+import Foundation
+import SwiftUI
+
+private struct PaletteBorder: ViewModifier {
```

Comment:
This needs to be applied to the crosspost list in `PostPage`

Reply:
Nice catch!